### PR TITLE
Boiler plate

### PR DESCRIPTION
**Pull request by abiodun0**

### CODE REVIEW

File: src/helpers/ApiClient.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        }
+
+        if (data) {
+          request.send(data);
+        }
+
+        request.end((err, { body } = {}) => err ? reject(body || err) : resolve(body));
+      }));
+  }
+  /*
```
Comment (atemitope) [inline src/helpers/ApiClient.js:37]: Do you think its necessary to leave the comment on the file?

File: src/theme/font-awesome.config.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/**
+ * Configuration file for font-awesome-webpack
+ *
+ * In order to keep the bundle size low in production,
+ * disable components you don't use.
```
Comment (atemitope) [inline src/theme/font-awesome.config.js:5]: Any reason why we want to keep this comment?

File: src/server.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  server.listen(config.port, (err) => {
+    if (err) {
+      console.error(err);
+    }
+    console.info('----\n==> ✅  %s is running, talking to API server on %s.', config.app.title, config.apiPort);
+    console.info('==> 💻  Open http://%s:%s in a browser to view the app.', config.host, config.port);
+  });
+} else {
+  console.error('==>     ERROR: No PORT environment variable has been specified');
+}
```
Comment (atemitope) [inline src/server.js:123]: Good Work on the proxy error handling

File: src/helpers/ApiClient.js (hunk 1 above)
Comment (abiodun0) [inline src/helpers/ApiClient.js:37]: Would remove this and update the changes

File: src/theme/font-awesome.config.js (hunk 2 above)
Comment (abiodun0) [inline src/theme/font-awesome.config.js:5]: So has to let the other developers working on this project to know a little bit indept about the config that goes into the webpack bootstrap css loader

**Pull request merged**